clio-server/src/main/scala/org/broadinstitute/clio/server/webservice/ReadGroupWebService.scala

```diff
@@ -29,12 +31,10 @@ trait ReadGroupWebService {
 
   private[webservice] val postMetadata: Route = {
     pathPrefix("metadata") {
-      pathPrefix("v1") {
-        pathPrefixKeyV1 { key =>
-          post {
-            entity(as[TransferReadGroupV1Metadata]) { metadata =>
-              complete(readGroupService.upsertMetadata(key, metadata))
-            }
+      pathPrefixKeyV1 { key =>
```

Comment:
Ah, I missed that it's still tied to the v1 transfer class. I think it's OK as-is then.